storybookjs/storybook
# Addon-docs: Dynamic source rendering for Vue

Issue: #11400 

## What I did

Added a decorator takes Vue story and emits source code for that.

NOTE: I didn't test directives (e.g. `v-model`, `v-on`, custom directives)

## How to test

- Is this testable with Jest or Chromatic screenshots? ... yes
- Does this need a new example in the kitchen sink apps? ... no
- Does this need an update to the documentation? ... no?

If your answer is yes to any of these, please make sure to include it in your PR. ... done (`addons/docs/src/frameworks/vue/sourceDecorator.test.ts`)



To see, run `cd examples/vue-kitchen-sink && yarn storybook`
